pypika/dialects.py

```diff
@@ -695,17 +713,6 @@ def top(self, value: Union[str, int], percent: bool = False, with_ties: bool = F
         self._top_percent: bool = percent
         self._top_with_ties: bool = with_ties
 
-    @builder
-    def fetch_next(self, limit: int) -> "MSSQLQueryBuilder":
-        # Overridden to provide a more domain-specific API for T-SQL users
-        self._limit = limit
```

Comment:
I see this doesn't get put into the `FetchNextAndOffsetRowsQueryBuilder` class. It might be good to include and put a deprecation warning if it will be removed in the future.

Reply:
Added a deprecation warning in the event `fetch_next` is used, and added a unittest.